skin.json

```diff
@@ -57,7 +61,12 @@
 				"assets/scripts/foundation/foundation.alert.js",
 				"assets/scripts/foundation/foundation.accordion.js",
 				"assets/scripts/foundation/foundation.abide.js",
-				"foreground/assets/scripts/foreground.js"
```

Comment:
I think we need to have a discussing about how many of the Foundation js functions to load. Things like abide, Magellan, Sliders, etc. If the skin isn't using them, can the user use them on pages? Magellan is cool, but how can a user use it on their wiki page? Most require markup which will be sanitized unless the wiki admin allows raw html.